rpc/block_svc_routines.c

```diff
@@ -4828,8 +4828,8 @@ block_delete_cli_1_svc_st(blockDeleteCli *blk, struct svc_req *rqstp)
     blockDecPrioAttr(glfs, blk->volume, info->prio_path);
   }
 
-  LOG("mgmt", GB_LOG_INFO, "delete cli returns success, block volume: %s/%s",
-      blk->volume, blk->block_name);
+  LOG("mgmt", GB_LOG_INFO, "delete cli return %s, block volume: %s/%s",
+      errCode ? "failure" : "success", blk->volume, blk->block_name);
```

Comment:
@lxbsz please also consider doing this for block_modify_cli_1_svc_st, block_modify_size_cli_1_svc_st and block_gen_config_cli_1_svc_st as well.

Thanks!

Reply:
Yeah, have already checked them all, they are all working well like this.

Thanks.

Reply:
@lxbsz I get what you are saying. But should this be printed in both success and failure cases for all the cli routines block_modify_cli_1_svc_st, block_modify_size_cli_1_svc_st and block_gen_config_cli_1_svc_st, block_replace_cli_1_svc_st, block_create_cli_1_svc_st, block_delete_cli_1_svc_st ?

May be this Logline should be moved to last line (or last goto label) of every function or just before return ? You get what I mean now ?

Reply:
I will fix all the others and make all the logs the same.

To move the logline to the last goto label, sometime there will be a problem, such as:

```
1089   /* Check Point */
1090   errCode = glusterBlockAuditRequest(glfs, blk, &cobj, list, &savereply);
1091   if (errCode) {
1092     LOG("mgmt", GB_LOG_ERROR, "glusterBlockAuditRequest: return %d"
1093         " volume: %s hosts: %s blockname %s", errCode,
1094         blk->volume, blk->block_hosts, blk->block_name); 
1095   } else if (!resultCaps[GB_CREATE_LOAD_BALANCE_CAP] && cobj.prio_path[0]) {
1096     blockIncPrioAttr(glfs, blk->volume, cobj.prio_path);
1097   }
1098 
1099   LOG("mgmt", GB_LOG_INFO, "create cli return %s, block volume: %s/%s",
1100       errCode ? "failure" : "success", blk->volume, blk->block_name);
1101 
1102  exist:
1103   GB_METAUNLOCK(lkfd, blk->volume, errCode, errMsg);  
``` 

The Line#1103, in the GB_METAUNLOCK macro, it will be possibly change the errCode too.

Thanks.

Reply:
@pkalever 
Ignore this, since we need to record the errCode even from the macro.
So I will fix this too.
Thanks.